CLAUDE.md

```diff
@@ -243,3 +248,5 @@ When asked about content from the HolmesGPT documentation website (https://robus
 - Kubernetes deployment: `docs/installation/kubernetes-installation.md`
 - Toolset documentation: `docs/data-sources/builtin-toolsets/`
 - API reference: `docs/reference/`
+
```

Comment:
@aantn Did you miss adding something here?

Reply:
@pavangudiwada wdym?

Reply:
Ther's an empty code block. Never mind. Please merge
<img width="1344" height="362" alt="CleanShot 2025-07-31 at 13 18 26" src="https://github.com/user-attachments/assets/9ed8073c-66fc-4e00-810b-2fe79a8efeac" />